app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,79 @@
+class ProjectsController < ApplicationController
+  before_action :set_project, only: [:show, :edit, :update, :destroy]
+  before_action :validates_role, only: [:new, :edit, :update, :create]
+  before_action :validates_admin, only: :destroy
+
+  def index
+    @projects = Project.all
```

Comment:
check the queries on this index and report back whats wrong.

Reply:
It executes three queries, one for user, second for client and third for project.